authtools/views.py

```diff
@@ -61,8 +64,8 @@ class WithNextUrlMixin(object):
     success_url = None
 
     def get_next_url(self):
-        if self.redirect_field_name in self.request.REQUEST:
-            redirect_to = self.request.REQUEST[self.redirect_field_name]
+        if self.redirect_field_name in self.request.GET:
+            redirect_to = self.request.GET[self.redirect_field_name]
```

Comment:
I don't think this is right. What if we have a next field on a form that we POST?